recipe/meta.yaml

```diff
@@ -1,6 +1,7 @@
 {% set name = "cuda-nvcc-impl" %}
-{% set version = "12.1.105" %}
-{% set cuda_version = "12.1" %}
+{% set version = "12.2.140" %}
+{% set cuda_version = "12.2" %}
+{% set cuda_version_next = cuda_version.split(".")[0]|int + 1 %}
```

Comment:
It might help to indicate this is the next _major_ version. Also we probably want it to look like `13.0a0` instead of `13` to align with how the `pin_compatible` function would specify the upper bound.
```suggestion
{% set cuda_version_next_major = (cuda_version.split(".")[0]|int + 1)|str + ".0a0" %}
```

Reply:
Renamed in https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/8/commits/047d00dd061b023ed4e7eb747ac937da20d2f639
Can't seem to get it working with the `|str + ".0a0"` bit right now though
```
Error: Failed to render jinja template in /home/conda/recipe_root/meta.yaml:
No filter named 'str'.
```

Reply:
Think it is called [`string`]( https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.string )

Reply:
Yup that was it, thanks!

Reply:
Thanks - I always forget that it's `string` and not `str`.

Reply:
Yeah I forgot the same thing just the other day. Hence why it is fresh in my brain 😂